src/material/slider/slider-thumb.ts

```diff
@@ -118,6 +118,7 @@ export class MatSliderVisualThumb implements _MatSliderVisualThumb, AfterViewIni
       input.addEventListener('pointerleave', this._onMouseLeave);
       input.addEventListener('focus', this._onFocus);
       input.addEventListener('blur', this._onBlur);
+      input.addEventListener('contextmenu', this._onDragEnd);
```

Comment:
Now that it's handled in `pointerdown`, we should be able to remove this listener.